packages/camera/camera_web/example/integration_test/camera_web_test.dart

```diff
@@ -527,10 +527,13 @@ void main() {
                 .mapResolutionPresetToSize(ResolutionPreset.ultraHigh),
           ).thenReturn(ultraHighResolutionSize);
 
-          final int cameraId = await CameraPlatform.instance.createCamera(
+          final int cameraId =
+              await CameraPlatform.instance.createCameraWithSettings(
```

Comment:
Please, ensure that `createCamera` retains tests.

I don't mind if you leave the original `createCamera` tests as they were, and you add the copies to test `createCameraWithSettings`.

Thanks!

Reply:
`createCameraWithSettings` tests created as copies.
also integration test for video bitrate is added.